includes/internal/emails/class-email-customization.php

```diff
@@ -179,14 +186,33 @@ public function init(): void {
 	 * @access private
 	 */
 	public function disable_legacy_emails() {
-		remove_action( 'sensei_course_status_updated', [ Sensei()->emails, 'teacher_completed_course' ] );
-		remove_action( 'sensei_user_course_start', [ Sensei()->emails, 'teacher_started_course' ] );
-		remove_action( 'sensei_user_quiz_submitted', [ Sensei()->emails, 'teacher_quiz_submitted' ] );
-		remove_action( 'sensei_course_status_updated', [ Sensei()->emails, 'learner_completed_course' ] );
-		remove_action( 'sensei_course_new_teacher_assigned', [ Sensei()->teacher, 'teacher_course_assigned_notification' ] );
-		remove_action( 'sensei_user_lesson_end', [ Sensei()->emails, 'teacher_completed_lesson' ] );
-		remove_action( 'sensei_user_quiz_grade', [ Sensei()->emails, 'learner_graded_quiz' ] );
-		remove_action( 'sensei_private_message_reply', [ Sensei()->emails, 'new_message_reply' ] );
-		remove_action( 'sensei_new_private_message', [ Sensei()->emails, 'teacher_new_message' ] );
+		$emails_to_be_disabled = [
```

Comment:
I think it is the kind of case making it a more simple is better because we are adding more code than just adding a new line.

Reply:
Thanks, I've reverted the change [here](https://github.com/Automattic/sensei/pull/6617/commits/6b0834aa4ef34cb89a4058f051446cadbd8aa51b)